lib/postgresql.js

```diff
@@ -92,7 +92,10 @@ function PostgreSQL(postgresql, settings) {
   }
 
   this.settings = settings;
-  debug('Settings %j', settings);
+  debug('Settings %j', {
+    ...settings,
+    ...(typeof settings.password !== 'undefined' && {password: 'HIDDEN'}),
```

Comment:
suggestion: Take into consideration `url` and use asterisks instead, which is more universally recognisable and language-agnostic.

```suggestion
    ...(typeof settings.password !== 'undefined' && {url: settings.url.replace(settings.password, '***') password: '***'}),
```

Reply:
If a URL is defined, it is not necessary to define a password outside of it. Therefore this would not cover it 100%. (See my last change on the commit)

Reply:
Example:
Url Connection: _postgresql://other:**secret**@localhost/otherdb?connect_timeout=10&application_name=myapp_
After Url Connection: _postgresql://other:***@localhost/otherdb?connect_timeout=10&application_name=myapp_

Reply:
You're right. Initially I though Juggler's `DataSource` would populate `settings` in both of these syntax:

```javascript
const {DataSource} = require('loopack-datasource-juggler');
// "Single URL" syntax
new DataSource('mssql://user:pass@127.0.0.1');
// "`url` settings" syntax
new DataSource({url: 'mssql://user:pass@127.0.0.1');
```

But apparently the "`url` settings" syntax does not cause the other settings fields to be populated (e.g.`password`), only the "Single URL" syntax.